frontend/admin/src/js/lang/fr.json

```diff
@@ -1664,5 +1664,599 @@
+  },
+  "VlI1K4": {
+    "defaultMessage": "Tous les utilisateurs",
+    "description": "Title for the admin users table"
+  },
+  "Vpk+nl": {
+    "defaultMessage": "Professionnel",
+    "description": "Title for pool advertisement occupational skill list"
+  },
+  "XprYVQ": {
+    "defaultMessage": "Rédiger l'introduction de cette offre d'emploi...",
+    "description": "Placeholder message for  the English - Your Impact textarea in the edit pool page."
+  },
+  "Xsxvql": {
+    "defaultMessage": "Saississez les tâches clés pour cette affiche de poste...",
+    "description": "Placeholder message for the French - Your Work textarea in the edit pool page."
+  },
+  "YH6bFU": {
+    "defaultMessage": "Colonnes du tableau",
+    "description": "Dialog title for the admin tables columns toggle."
+  },
+  "aGI59+": {
+    "defaultMessage": "Ces informations permettent aux candidats de savoir à quel type de travail et pour quel environnement ils postulent. Utilisez cet espace pour parler du domaine du gouvernement que ce processus visera à améliorer. Ainsi que la valeur que crée ce type de travail",
+    "description": "Helper message for filling in the pool introduction"
+  },
+  "bbIDc9": {
+    "defaultMessage": "Enregistrer le nom du bassin",
+    "description": "Text on a button to save the pool name"
+  },
+  "cEeW3m": {
+    "defaultMessage": "Emplacement (français) : {locationFr}",
+    "description": "Pool advertisement location requirement, French"
+  },
+  "cQE6qE": {
+    "defaultMessage": "Rédiger l'introduction de cette offre d'emploi...",
+    "description": "Placeholder message for the French - Your Impact textarea in the edit pool page."
+  },
+  "chFoB8": {
+    "defaultMessage": "Tableau de recherche",
+    "description": "Label for search field on admin tables."
+  },
+  "cy5aj8": {
+    "defaultMessage": "État",
+    "description": "Label for a pool advertisements status"
+  },
+  "dG1EJO": {
+    "defaultMessage": "Ce bassin est annoncé, mais n'accepte plus de candidatures.",
+    "description": "Status description of a pool in EXPIRED status"
+  },
+  "dJxNRU": {
+    "defaultMessage": "Annuler et revenir en arrière",
+    "description": "Label displayed on cancel button for new pool form."
+  },
+  "de2F/x": {
+    "defaultMessage": "Annonce du bassin",
+    "description": "Label for pool advertisement url field"
+  },
+  "dmGvCL": {
+    "defaultMessage": "Modifier l'annonce du bassin",
+    "description": "Link text for button to edit a specific pool"
+  },
+  "eGEusj": {
+    "defaultMessage": "Transférable",
+    "description": "Title for pool advertisement transferable skill list"
+  },
+  "ekGCv2": {
+    "defaultMessage": "Cela mettra votre bassin à la disposition des candidats pour qu'ils puissent soumettre des candidatures",
+    "description": "Second paragraph for publish pool dialog"
+  },
+  "fOCbBa": {
+    "defaultMessage": "Résultats des compétences générales"
```

Comment:
Is it possible to add a description to the English string so that one can find the English string if necessary?